### ISSUE TITLE
 Invalid configuration `wasm32-unknown-emscripten': system `emscripten' not recognized

### ISSUE THREAD

**Issue by umdreamer** [emscripten-core/emscripten#22676]
I'm building Python 3.8.10 using emscripten. I used the same command line like https://rdb.name/panda3d-webgl.md. But these error accured.
`READELF=true BASECFLAGS="-s FILESYSTEM=0 -s DISABLE_EXCEPTION_THROWING=1 -s DISABLE_EXCEPTION_CATCHING=1" LDFLAGS="-s USE_ZLIB=1 -s DISABLE_EXCEPTION_THROWING=1 -s DISABLE_EXCEPTION_CATCHING=1" emconfigure ./configure --host=wasm32-unknown-emscripten --without-pymalloc --disable-shared --enable-ipv6 --prefix= --build=x86_64-pc-linux-gnu --enable-optimizations configure: ./configure --host=wasm32-unknown-emscripten --without-pymalloc --disable-shared --enable-ipv6 --prefix= --build=x86_64-pc-linux-gnu --enable-optimizations`

```
checking build system type... x86_64-pc-linux-gnu
checking host system type... Invalid configuration `wasm32-unknown-emscripten': system `emscripten' not recognized
configure: error: /bin/bash ./config.sub wasm32-unknown-emscripten failed
emconfigure: error: './configure --host=wasm32-unknown-emscripten --without-pymalloc --disable-shared --enable-ipv6 --prefix= --build=x86_64-pc-linux-gnu --enable-optimizations' failed (returned 1)
```
Does anyone help? Thanks.

**Comment by floooh** [comment]
That looks more like a bug in the Panda3D build process tbh. Did you ask around on the Panda Discord as recommended in the blog post?

**Comment by umdreamer** [comment]
Yes, I have asked similar questions in Panda3D discourse. I have not got response. Maybe later there will be some reply. I found another thing is that Python 3.11 and 3.12 have the `wasm32-unknown-emscripten`, but the emscripten support is dropped in Python 3.13. Python 3.8 doesn't support `wasm32-unkown-emscripten` yet. I tried to modify the Python 3.8's configure.ac file using Python 3.11, but it doesn't work either.

**Comment by sbc100** [comment]
Can you try building without the `--host=wasm32-unknown-emscripten`.   IIUC emconfigure tried to pretend its not a cross compiler.
